app/views/overview.html

```diff
@@ -12,38 +12,9 @@
 
 {% include "includes/sub-nav.html" %}
 
-<div class="grid-row">
-  <div class="column-two-thirds">
-    <table>
-    <tbody>
-        <tr>
-        <th scope="row">Grade</th>
-        <td class="sln-grade numeric">{{ overviewDetails.grade }}</td>
-        </tr>
-        <tr>
-        <th scope="row">Team</th>
-        <td class="numeric"><a href="/team/{{ overviewDetails.teamId }}">{{ overviewDetails.teamName }}</a></td>
-        </tr>
-        <tr>
-        <th scope="row">Capacity</th>
-        <td class="numeric"> {{ overviewDetails.capacity | round }}%</td>
-        </tr>
-        <tr>
-        <th scope="row">Cases</th>
-        <td class="numeric">{{ overviewDetails.cases }}</td>
-        </tr>
-        <tr>
-        <th scope="row">Contracted hours</th>
-        <td class="numeric">{{ overviewDetails.contractedHours }}</td>
-        </tr>
-        <tr>
-        <th scope="row">Reduction</th>
-        <td class="numeric">{{ overviewDetails.reduction }}</td>
-        </tr>
-    </tbody>
-    </table>
-  </div>
-</div>
+{% include "includes/individual-overview.html" %}
+
+{% include "includes/team-overview-table.html" %}
```

Comment:
Yea, so part of the next story, we are going to refactor that to org unit overview (or something like that).

We tried the if/case statement, but the way in which nunjucks renders the templates, it appears to render the includes before it performs the if statements. There is an if statement  at the top of the include files to cover this.